### PR TITLE
feat(GLobalVariables) adding the Application_ListView_FilterPanel_Open

### PR DESCRIPTION
**Pull request by maliknajjar**
We need to create a global variable (Application_ListView_FilterPanel_Open) that takes [ 0 | 1 ] as values.
When the GV is set to 0, it should:

* hide thee filter's panel as shown in the image below:

### CODE REVIEW

File: modules/Vtiger/ListView.php [hunk 1]
```diff
@@ -272,6 +272,12 @@
 }
 $smarty->assign('advancedSearchMapResult', $advancedSearchMapResult);
 
+// GV responsible for showing or hiding filter panel
+$Application_ListView_FilterPanel_Open = GlobalVariable::getVariable('Application_ListView_FilterPanel_Open', '1', $currentModule, '', $_REQUEST['action']);
+if (!$Application_ListView_FilterPanel_Open) {
+	$smarty->assign('Application_ListView_FilterPanel_Open', "display: none;");
+}
```
Comment (joebordes) [inline modules/Vtiger/ListView.php:279]: you need an "else" here to set the Smarty variable in the case it is not open

File: modules/Vtiger/ListView.php [hunk 2]
```diff
@@ -272,6 +272,12 @@
 }
 $smarty->assign('advancedSearchMapResult', $advancedSearchMapResult);
 
+// GV responsible for showing or hiding filter panel
+$Application_ListView_FilterPanel_Open = GlobalVariable::getVariable('Application_ListView_FilterPanel_Open', '1', $currentModule, '', $_REQUEST['action']);
```
Comment (joebordes) [inline modules/Vtiger/ListView.php:276]: I think the logic is inverted here
the default value is "1! but in that case you set the smarty variable to "display none"  
Is that correct?

Comment (maliknajjar) [reply modules/Vtiger/ListView.php:276]: I set it to display none if it is not "1" `!$Application_ListView_FilterPanel_Open`

File: modules/Vtiger/ListView.php (hunk 1 above)
Comment (maliknajjar) [reply modules/Vtiger/ListView.php:279]: the reason why I kept it like that is because Smarty did not crash .. but I will change it right away

**Comment by joebordes** [comment]
thanks!!

**Pull request merged**